packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md

```diff
@@ -0,0 +1,76 @@
+## Integration steps
+
+### Step 1: User launches the sign-in page
+
+Build a sign-in page for your app that captures both the username and the password.
+
+<div class="common-image-format">
+
+![Displays the Java SDK sign-in page](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in-java.png)
+
+</div>
+
+Begin the authentication process by calling Java SDK's [`IDXAuthenticationWrapper.begin()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L603) method and getting a new [`ProceedContext`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/ProceedContext.java) object.
+
+```kotlin
+val beginResponse = idxAuthenticationWrapper.begin()
+val proceedContext = beginResponse.proceedContext
+```
+
+### Step 2: User enters their credentials
+
+After the user submits their credentials, call `IDXAuthenticationWrapper.authenticate()` with the credential values.
+
+```kotlin
+val authenticationResponse =
+     idxAuthenticationWrapper.authenticate(AuthenticationOptions(username, password)), proceedContext)
+```
+
+If the password is validated, the `IDXAuthenticationWrapper.authenticate()` method returns an [`AuthenticationResponse`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java) object with the following:
+
+1. `AuthenticationStatus` = `AWAITING_AUTHENTICATOR_SELECTION` <br>
```

Comment:
```suggestion
* `AuthenticationStatus= AWAITING_AUTHENTICATOR_SELECTION` &mdash; This status indicates that there are required authenticators that need to be verified.
```

Reply:
This formatting is just a suggestion. I've implemented the bullet/em-dash convention after the password/phone use case review with Chris (and have yet to fix all my previous use cases).